plugins/tile/tile-plugin.cpp

```diff
@@ -271,28 +289,58 @@ class tile_output_plugin_t : public wf::pointer_interaction_t, public wf::custom
         });
     };
 
+    wf::key_callback on_toggle_maximized = [=] (auto)
+    {
+        auto view = wf::get_core().seat->get_active_view();
```

Comment:
You should use the `conditioned_view_execute` helper function here (see how the other functions use it). This ensures the binding is not toggled while another plugin is active.